idd/Energy+.idd.in

```diff
@@ -106278,3 +106278,7 @@ PythonPlugin:OutputVariable,
        \note Enter a user-defined subcategory for this metered output
        \type alpha
        \retaincase
+
+EnhancedInteractivity;
```

Comment:
This capability is enabled through an input object being in the IDF.  The object requires no extra fields just the name.